pandas/core/generic.py

```diff
@@ -5822,20 +5822,24 @@ def _where(self, cond, other=np.nan, inplace=False, axis=None, level=None,
 
```

Comment:
can you update same for ``.mask`` as well.

Reply:
mask and where now use the same docstring (with only parts of it substituted with the correct value). But keep doing this and providing a better explanation will become rather complex I think. Are you OK with just duplicating the docstring and have one for where and mask separately ?

Reply:
oh that's right. I don't think duplicating is a good idea. maybe just put an example with mask here as well.